test/cpp/nativert/test_conv.cpp

```diff
@@ -0,0 +1,31 @@
+#include <gtest/gtest.h>
+#include <torch/nativert/common/Conv.h>
+
+namespace torch::nativert {
+
+TEST(TryToTest, Int64T) {
+  // Test valid conversions
+  EXPECT_EQ(tryTo<int64_t>("123"), 123);
+  EXPECT_EQ(tryTo<int64_t>("456"), 456);
+  // Test invalid conversions
+  EXPECT_FALSE(tryTo<int64_t>("0x123").has_value());
+  EXPECT_FALSE(tryTo<int64_t>("123abc").has_value());
+  EXPECT_FALSE(tryTo<int64_t>("123.45").has_value());
+  EXPECT_FALSE(tryTo<int64_t>("").has_value());
+  // Test overflow
+  EXPECT_FALSE(tryTo<int64_t>("12345678901234567890").has_value());
```

Comment:
you could make this less verbose by putting the test strings in a vector and looping:

```
const std::vector<const char*> invalid_examples = {
  "0x123",
  "123abc",
  ...
};
for (const char* bad : invalid_examples) {
  EXPECT_FALSE(tryTo<int64_t>(bad).has_value());
}
```

(and similarly for the valid examples and other test cases)